cli/command/network/connect.go

```diff
@@ -37,11 +37,11 @@ func newConnectCommand(dockerCli *command.DockerCli) *cobra.Command {
 	}
 
 	flags := cmd.Flags()
-	flags.StringVar(&opts.ipaddress, "ip", "", "IP Address")
-	flags.StringVar(&opts.ipv6address, "ip6", "", "IPv6 Address")
+	flags.StringVar(&opts.ipaddress, "ip", "", "IPv4 address (e.g. 172.30.100.104)")
+	flags.StringVar(&opts.ipv6address, "ip6", "", "IPv6 address (e.g. 2001:db8::33)")
 	flags.Var(&opts.links, "link", "Add link to another container")
 	flags.StringSliceVar(&opts.aliases, "alias", []string{}, "Add network-scoped alias for the container")
-	flags.StringSliceVar(&opts.linklocalips, "link-local-ip", []string{}, "Add a link-local address for the container")
+	flags.StringSliceVar(&opts.linklocalips, "link-local-ip", []string{}, "Container IPv4/IPv6 link-local addresses")
```

Comment:
I think I prefer the `Add a link-local ...` form here. Using the plural "addresses" may lead to think that you can provide multiple addresses on a single flag. "Add a " is _just_ a bit more clear that the flag expects a single address (but can be provided multiple times).

Alternatively, just `IPv4/IPv6 link-local address`

(But suggestions are welcome here; it's difficult to express that a flag can be provided multiple times in a very short description)